shared/stats/rollup.config.mjs

```diff
@@ -0,0 +1,36 @@
+import { join } from 'path'
+
+import configESM from 'govuk-frontend/rollup.esm.config.mjs'
```

Comment:
Wondering if we do need to reuse the configs from govuk-frontend. It feels a risk to impact the stats when we update the config for building govuk-frontend. Do the stats need much more in terms of config than having the visualiser plugin?

Reply:
Ahh I figured this would reduce risk?

For example when we add `@rollup/plugin-babel` to **rollup.esm.config.mjs** the stats will change

In the PR preview deployment we'll instantly be able to compare them

Reply:
Wouldn't we be running `babel` twice if that's the case: once to build `govuk-frontend/dist` and then that one going through `babel` again when building the stats, which is not what we're looking for?

Reply:
Nah I'd rather we only run it once. It won't run on `node_modules` unless configured to

Reply:
Yeah, I'd rather it only runs once when we build `govuk-frontend` as well and sounds like by default it'll only run like that. That's just for Babel though, other plugins will likely work at all times. The reason I'm advocating for not reusing the `govuk-frontend` configs there it to avoid risks of a change in `govuk-frontend` configuration changing our stats computation. 

Framing the question differently: what do we gain by re-using the `govuk-frontend` rollup configs?

Reply:
Just went for the fastest option that didn't duplicate code. These configs are all ready to go

Shall we hold off setting up more configs until it becomes an issue?

Reply:
I think in this case we do need the duplication to make sure things are independent. By sharing the configs, we're mixing, in the computation of stats, stuff that we run for the build of the package itself (like the replacement of the version). While it doesn't do affect it much for now, that's a risk in the future to double up some transformations without noticing or maybe make multiple outputs for our build (which I'm not sure how the stats plugin will cope with, thinking of it).

At the end of the day, the only part we're getting from that sharing is the config, so I'd rather we start with a clean separation from the get go there. I think this will also avoid the need for updating the current Rollup configs as well which is that less changes 😊

Reply:
Pushed up an update. All configs are now standalone